### PR TITLE
Render Rework Fase 1

### PR DESCRIPTION
**Pull request by rosaqq**
Currently in progress.

### CODE REVIEW

**Comment by rosaqq** [comment]
Turns out SkyBox is actually a GameItemMesh but with a different render implementation:

image

**Comment by rosaqq** [comment]
Collider not yet implemented, entao esse boundix box behaviour ta no GameItemMesh. Mas o HudElement is also GameItemMesh so cook code to keep engine compiling and not try to use HudElements in physics engine...

Hud is broken tho...
Acho que vai ficar fixed com esse collider class.

**Comment by dffsa** [comment]
Já experimentaste fazer savescene e loadscene para ver se esse serialization code não ficou trolled

File: src/main/java/net/sknv/engine/GameEngine.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 public class GameEngine implements Runnable {
 
     public static final int TARGET_FPS = 144;
-    public static final int TARGET_UPS = 200;
+    public static final int TARGET_UPS = 60;
```
Comment (dffsa) [inline src/main/java/net/sknv/engine/GameEngine.java:9]: pls 200

**Pull request merged**